www-servers/tomcat/tomcat-10.1.31.ebuild

```diff
@@ -9,7 +9,7 @@ inherit java-pkg-2 prefix verify-sig
 
 MY_P="apache-${P}-src"
 
-DESCRIPTION="Tomcat Servlet-6.0/JSP-3.1/EL-5.0/WebSocket-2.1/JASPIC-3.0 Container"
+DESCRIPTION="Servlet-6.0/JSP-3.1/EL-5.0/WebSocket-2.1/JASPIC-3.0 Container"
```

Comment:
i'm not sure about these changes. maybe it would be better to state it similarly as on their homepage: `Apache Tomcat 10.1 is an implementation of Servlet-6.0/JSP-3.1/EL-5.0/WebSocket-2.1/JASPIC-3.0`. or if we use the same descriptions for all the ebuilds, then like this: `Apache Tomcat is an implementation of the Jakarta Servlet, Jakarta Pages, Jakarta Expression Language, Jakarta WebSocket, Jakarta Annotations and Jakarta Authentication specifications.` - but probably using abbreviations of the specifications where possible to be within the limit of description length.

so, to sum it up, we either use a generic description without the versions, or (i think) we need to use the version in te description as only that major version implements the specified specification versions.

Reply:
> i'm not sure about these changes. 

Indeed, if we want to do it correctly it becomes difficult. Let's keep it as is. I will drop that commit.
The more interesting question for me is: Should we try to unbundle `jakarta-annotation-api` and `jakarta-servlet-api` and possibly more?

Reply:
that should be imo doable as that's the standard api and should be replaceable.

Reply:
> that should be imo doable as that's the standard api and should be replaceable.

Will see if it's possible without too complex patching.